persistence/src/main/scala/za/co/absa/spline/persistence/ArangoRepoConfig.scala

```diff
@@ -33,7 +34,13 @@ class ArangoRepoConfig extends InitializingBean with Logging {
     log.info(s"Spline database URL: ${Database.ConnectionURL.asString}")
   }
 
-  @Bean def arangoDatabaseFacade: ArangoDatabaseFacade = new ArangoDatabaseFacade(Database.ConnectionURL)
+  @Bean def arangoDatabaseFacade: ArangoDatabaseFacade = {
+    val sslCtxOpt =
+      if (!Database.DisableSSLValidation) None
+      else Some(TLSUtils.TrustingAllSSLContext)
```

Comment:
Unnecessary negation

Reply:
OK, will flap it.
The reason why I added extra negation in those two `if`'s was just keep code lines shorter.

Reply:
I've added `Option` builders from Scala 13 and used them instead of `if-else` block. I think it reads even better.
```scala
val sslCtxOpt = Option.when(conf.disableSslValidation)(TLSUtils.TrustingAllSSLContext)
```